unit flake: TestDQ_sanity_check is flaky

Failure:

    --- FAIL: TestDQ_sanity_check (2.01s)
	delay_test.go:137: delay too short: 1.881104721s, expected: 2s
    --- PASS: TestDFIFO_sanity_check (2.06s)
    --- PASS: TestDQ_ordered_add_pop (3.13s)
    FAIL
    FAIL	k8s.io/kubernetes/contrib/mesos/pkg/queue	3.453s

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/25344/kubernetes-pull-test-unit-integration/24932/build-log.txt

--- a/contrib/mesos/pkg/queue/delay_test.go
+++ b/contrib/mesos/pkg/queue/delay_test.go
@@ -125,9 +125,9 @@ func TestDQ_sanity_check(t *testing.T) {
 
 	dq := NewDelayQueue()
 	delay := 2 * time.Second
-	dq.Add(&testjob{d: delay})
 
 	before := time.Now()
+	dq.Add(&testjob{d: delay})
 	x := dq.Pop()
 
 	now := time.Now()

